pkg/azure/aks/machine_store.go

```diff
@@ -358,12 +368,12 @@ func (m *MachineStore) PopulateMachineStore(ctx context.Context) error {
 
 	err = eg.Wait()
 	if err != nil {
-		return err
+		m.logger.Error("BIG ERROR YO")
```

Comment:
heh, love the intent, but let's update this error message with what the problem is.

Reply:
Updated. The shame 🤦